### PR TITLE
Db setup

### PR DESCRIPTION
**Pull request by Chenxi96**
## Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] I have added thorough tests. (If tests are not applicable, explain why in the next section.)

## Issue ticket number and link
N/A
## Describe your changes
I added Prisma files and also fixed the deployment issue.

### CODE REVIEW

**Comment by alvinc21** [comment]
Nice its passing now! Squash the commits and this PR will be all set.

File: package.json [hunk 1]
```diff
@@ -11,14 +11,18 @@
...
   "license": "MIT",
   "dependencies": {
+    "@prisma/client": "^5.11.0",
     "next": "12.0.10",
+    "npm": "^10.5.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-markdown": "8.0.0"
+    "react-markdown": "8.0.0",
+    "vercel": "^33.6.2"
```
Comment (alvinc21) [inline package.json:20]: Wow this was the culprit? Good job finding it.

Comment (Chenxi96) [reply package.json:20]: yeah deleted the package-lock.json then npm installed again, after that it worked tbh I got lucky lol

**Review by Chenxi96 — COMMENTED** [review]
@alvinc21 let me know if I did the squashing right

**Review by alvinc21 — APPROVED** [review]
First PR 👏. Merge it when you're ready.

**Pull request merged**